pkg/entities/nfts.go

```diff
@@ -51,22 +51,26 @@ func (e *Entity) GetNFTDetail(symbol, tokenID string) (*response.IndexerNFTToken
 	collection, err := e.repo.NFTCollection.GetBySymbolorName(symbol)
 	// cannot find collection in db
 	if err != nil {
+		e.log.Errorf(err, "[repo.NFTCollection.GetBySymbolorName] failed to get nft collection by %s", symbol)
```

Comment:
"[repo.NFTCollection.GetBySymbolorName] failed to get nft collection by symbol: %s"